tests/utils/test_utils.go

```diff
@@ -0,0 +1,79 @@
+package testutils
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/rs/zerolog/log"
+)
+
+type EnvData struct {
+	Src string `json:"src"`
+	Val string `json:"val"`
+}
+
+func checkL2BinaryExists(l2BinPath string) error {
+	if _, err := os.Stat(l2BinPath); os.IsNotExist(err) {
+		return fmt.Errorf("l2 binary not found in the build folder %s, please change the path", l2BinPath)
+	}
+	return nil
+}
+
+func getL2BinaryPath() (string, error) {
```

Comment:
`getLocalL2BinaryPath` to differentiate from system binary.

Reply:
right